test/VS.Web.CG.Mvc.Test/IdentityGeneratorTemplateModelBuilderTests.cs

```diff
@@ -71,6 +72,45 @@ public IdentityGeneratorTemplateModelBuilderTests(ITestOutputHelper output)
             }
         }
 
+        private static readonly string LayoutFileLocationTestProjectBasePath = "c:\\users\\test\\source\\repos\\Test1\\";
+
+        // Tests for determining the support file location when an existing layout path is specified.
+        // The input layout file path is relative to the project root.
+        [Theory]
+        [InlineData("Views/Shared/Layout.cshtml", "Views\\Shared", "Views/Shared/Layout.cshtml")]
```

Comment:
Do these tests pass on non-windows as well? 
Might be better to use `Path.Combine()` for the `expectedSupportFileLocation`

Reply:
Good call - they fail on non windows.